DFC.ServiceTaxonomy.Taxonomies/Controllers/AdminController.cs

```diff
@@ -297,11 +300,10 @@ public async Task<IActionResult> EditPost(string taxonomyContentItemId, string t
             taxonomy.Published = false;
             await _contentManager.PublishAsync(taxonomy);
 
-            //force publish the new term content item to trigger events etc
-            await _contentManager.PublishAsync(contentItem);
-
+            //Content item will get published as part of the taxonomy, handler ensure Event Grid is informed of Content Item change
             foreach (var handler in _handlers)
             {
+                await handler.PublishedAsync(contentItem);
                 await handler.UpdatedAsync(contentItem, taxonomy);
```

Comment:
We need to handle errors that happen in `UpdatedAsync` better. If it can't update pages and cancels the session, then we shouldn't call `UpdatedAsync` so that events aren't published. We'd have to check that it also cancels the term change too.
Think it might need a story!